test/Object/getOwnPropertyDescriptors.js

```diff
@@ -0,0 +1,89 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+
+if (this.WScript && this.WScript.LoadScriptFile) { // Check for running in ch
+    this.WScript.LoadScriptFile("..\\UnitTestFramework\\UnitTestFramework.js");
+}
+
+var tests = [
+    {
+        name: "Object has getOwnPropertyDescriptors method",
+            body: function() {
+                assert.isTrue(Object.hasOwnProperty("getOwnPropertyDescriptors"), 'Object.hasOwnProperty("getOwnPropertyDescriptors")');
+

```

Comment:
Need more tests here for the spec details of the property and function object, i.e. the property attributes are set correctly and the length property of the function is correct.  There are examples of such tests in some of the other es6 tests.